start.sh

```diff
@@ -44,6 +44,27 @@ fi
 
 LOOPS=0
 
+handle_exit_code() {
+	local exitcode=$1
+	if [ "$exitcode" -eq 134 ] || [ "$exitcode" -eq 139 ]; then #SIGABRT/SIGSEGV
+		echo ""
+		echo "ERROR: The server process was killed due to a critical error (code $exitcode) which could indicate a problem with PHP."
+		echo "Updating your PHP binary is recommended."
+		echo "If this keeps happening, please open a bug report."
+		echo ""
```

Comment:
Segmentation fault which is due to plugin error could lead to a lot of bug report, no ?

Reply:
PHP code shouldn't be causing segfaults

Reply:
Segfault can be caused in PHP code only via third-party PHP extensions. There is an open issue with `ext-pmmpthread`, but plugin devs are not supposed to use it that way.

Reply:
> PHP code shouldn't be causing segfaults

If you make some tricky code in a thread causing segfault